src/Raven.Client/Documents/Session/IDocumentQueryBase.cs

```diff
@@ -482,265 +478,331 @@ public interface IDocumentQueryBase<T, TSelf> : IPagingDocumentQueryBase<T, TSel
         int Count(Func<T, bool> predicate);
 
         /// <summary>
-        ///     Apply distinct operation to this query
+        ///     Removes duplicates from query results.
```

Comment:
We have a dedicated article about Distinct:
https://ravendb.net/docs/article-page/6.0/csharp/indexes/querying/distinct
Why not to reference it?

Reply:
Added